<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommVisAddCommDialog</class>
 <widget class="QDialog" name="CommVisAddCommDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Comm Device</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QFormLayout" name="mFormLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="mPlatformNameLabel">
       <property name="text">
        <string>Platform Name: </string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="mPlatformNameComboBox">
       <item>
        <property name="text">
         <string>&lt;Choose Platform&gt;</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="mCommNameLabel">
       <property name="text">
        <string>Comm Name:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="mCommNameLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="mCommTypeLabel">
       <property name="text">
        <string>Comm Type:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="mCommTypeLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="mNetworkNameLabel">
       <property name="text">
        <string>Network Name:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="mNetworkNameLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>default</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="mNetworkAddressLabel">
       <property name="text">
        <string>Network Address:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="mNetworkAddressLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="mAddressLabel">
       <property name="text">
        <string>Address:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="mAddressLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="mLinkPlatformLabel">
       <property name="text">
        <string>Link Platform:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="mLinkPlatformLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="mLinkCommLabel">
       <property name="text">
        <string>Link Comm:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="mLinkCommLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="mLinkAddressLabel">
       <property name="text">
        <string>Link Address:</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="mLinkAddressLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="mLocalLinkLabel">
       <property name="text">
        <string>Local Link:</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLineEdit" name="mLocalLinkLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="mRouterNameLabel">
       <property name="text">
        <string>Router Name:</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLineEdit" name="mRouterNameLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>default</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="mGatewayPlatformLabel">
       <property name="text">
        <string>Gateway Platform:</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QLineEdit" name="mGatewayPlatformLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="mGatewayCommLabel">
       <property name="text">
        <string>Gateway Comm:</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QLineEdit" name="mGatewayCommLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mPlatformNameComboBox</tabstop>
  <tabstop>mCommNameLineEdit</tabstop>
  <tabstop>mCommTypeLineEdit</tabstop>
  <tabstop>mNetworkNameLineEdit</tabstop>
  <tabstop>mNetworkAddressLineEdit</tabstop>
  <tabstop>mAddressLineEdit</tabstop>
  <tabstop>mLinkPlatformLineEdit</tabstop>
  <tabstop>mLinkCommLineEdit</tabstop>
  <tabstop>mLinkAddressLineEdit</tabstop>
  <tabstop>mLocalLinkLineEdit</tabstop>
  <tabstop>mRouterNameLineEdit</tabstop>
  <tabstop>mGatewayPlatformLineEdit</tabstop>
  <tabstop>mGatewayCommLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CommVisAddCommDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CommVisAddCommDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
